checker/build.gradle

```diff
@@ -382,6 +382,12 @@ task jtregJdk11Tests(dependsOn: ':downloadJtreg', group: 'Verification') {
 
     }
 }
+
+task testAnnotatedJdk {
+    description 'Runs some of the tests that use the annotated jdk.'
```

Comment:
Is this only some of the tests that use the annotated JDK?  Or is this some of the tests -- all those that use the annotated JDK?

Reply:
Nit: capitalize JDK.